### PR TITLE
Tooltip: add new prop skipAndroidStatusBar for using inside RN Modal

### PR DESCRIPTION
**Pull request by oxyii**
Added new bool prop `skipAndroidStatusBar` with `false` as default value.

Related #1871

### CODE REVIEW

**Comment by iRoachie** [comment]
Really wish we could find another way around this 😞

**Comment by oxyii** [comment]
@iRoachie I'm afraid not. RN Modal sets `top=0` on first point **after** StatusBar. Tooltip cannot find out that it is in modal.

Выделение_001

Modal - it's next layer over main view layer.

Выделение_002

Please note that margins and paddings are not defined for modal content, but it does not fit under StatusBar

**Comment by oxyii** [comment]
@iRoachie Ready for re-review. Please help me with docs, my english is bad ((

**Comment by iRoachie** [comment]
Need to add it to the list of props here. You can follow a similar pattern as the others. https://github.com/react-native-training/react-native-elements/blame/master/docs/overlay.md#L59

**Comment by oxyii** [comment]
@iRoachie I had a strange error locally. IDE did not save my changes to disk, and they did not hit in git. Sorry, fixed.

**Comment by oxyii** [comment]
@iRoachie I rechecked. All commited now

**Comment by oxyii** [comment]
Looks also related #1888

**Comment by duydtd97** [comment]
yeahhh, thank you !  I have icons with tooltip in a RN modal and them not rendered properly and your gist work fine for me.

**Comment by iRoachie** [comment]
@oxyii Would this fix close https://github.com/react-native-training/react-native-elements/issues/1871?

**Comment by oxyii** [comment]
@iRoachie yes! And not only. Any tooltip vertical pos issue on android. This issue - it's android StatusBar issue again. Expo can handle StatusBar, `react-navigation`, `react-native-navigation` and more and more and more. We cannot detect all possibe ways. But we can allow users add StatusBar height (how it works now) and skip StatusBar height via new prop

**Comment by iRoachie** [comment]
Is this dependent on the discussion around react-native-status-bar-height?

**Comment by oxyii** [comment]
Not exactly but rel #1854. We can detect if expo handled StatusBar but we cannot detect if StatusBar handled by `react-navigation` or any other lib. Because of this new prop proposed

**Comment by dorthwein** [comment]
running into this as well...

**Comment by oxyii** [comment]
@iRoachie Let make this PR alive

File: docs/tooltip.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ import Modal from 'modal-react-native-web';
 - [`width`](#width)
 - [`withOverlay`](#withoverlay)
 - [`withPointer`](#withpointer)
+- [`skipAndroidStatusBar`](#skipAndroidStatusBar)
```
Comment (iRoachie) [inline docs/tooltip.md]:
```suggestion
- [`skipAndroidStatusBar`](#skipandroidstatusbar)
```

Comment (oxyii) [reply docs/tooltip.md]: Ohhh... Sure. Sorry

**Comment by iRoachie** [comment]
Thanks again @oxyii 🎉  Sorry took so long

**Comment by revathskumar** [comment]
@oxyii Thanks for fixing this.
@iRoachie This will get released in v2.0.0 or can we expect a minor release before v2?

**Comment by iRoachie** [comment]
@revathskumar Not before v2, not doing any other minors before then.

**Comment by revathskumar** [comment]
@iRoachie Is there any tentative timeline planned for v2 release?

**Comment by iRoachie** [comment]
First week in December

**Comment by vikjovanov** [comment]
Hi, 
When is this feature coming out ?

**Pull request merged**